.travis.yml

```diff
@@ -8,11 +8,15 @@ rvm:
   - 2.2
   - 2.3.3
   - 2.4.0
+  - ruby-head
 
 matrix:
   include:
     - rvm: 2.3.0
       script: bundle exec rake test_isolated
+  allow_failures:
+    - rvm: ruby-head
+  fast_finish: true
```

Comment:
Ah we do have those in Rails exactly.

In my experience to work for new version Ruby compatibility in another project, it was hard to explain new version Ruby's issue to other contributors.
Because they did not have the environment on their local for new version Ruby.
So I thought it is useful for us to share the issue on Travis easily.